en/vector-search.html

```diff
@@ -0,0 +1,248 @@
+  easier explained by example:
+</p>
+<p>
+  Consider a user searching for the flower "Dandelion".
+  A regular text search will match <a href="https://en.wikipedia.org/wiki/Taraxacum">this Wikipedia article</a>,
+  as the "dandelion" term is in the text:
+</p>
+<p>
+  <span style="font-style: italic; font-size: 1.8em; font-family: 'Linux Libertine','Georgia','Times',serif;">
+  Taraxacum</span>
+<p>
+<hr/>
+<p style="font-size: 92%; font-family: sans-serif;">
+  From Wikipedia, the free encyclopedia (Redirected from <span style="background-color: yellow;">Dandelion</span>)
+</p>
+<blockquote style="font-family: sans-serif;">
+  "<span style="background-color: yellow;">Dandelion</span>" redirects here.
+  It may refer to any species of the genus Taraxacum or specifically to Taraxacum officinale.
+  For similar plants, see False <span style="background-color: yellow;">dandelion</span>.
+  For other uses, see <span style="background-color: yellow;">Dandelion</span> (disambiguation)
+</blockquote>
+<p style="font-family: sans-serif;">
+  <strong><em>Taraxacum</em></strong> (/təˈræksəkʊm/) is a large genus of flowering plants in the family Asteraceae,
+  which consists of species commonly known as dandelions.
+</p>
+<p>
+  ...
+</p>
+<hr/>
+<p>
+  This document is about dandelions, for sure.
+  However, if the user searches for "blowball" (the mature spherical seed head of a dandelion),
+  the text above will not match, although the document is a good match for "blowball".
+  Example images of a dandelion:
+</p>
+<img src="https://upload.wikimedia.org/wikipedia/commons/4/4f/DandelionFlower.jpg"
+     alt="DandelionFlower.jpg" width="220" height="176" />
+<img src="https://upload.wikimedia.org/wikipedia/commons/5/54/TaraxacumOfficinaleSeed.JPG"
+     alt="DandelionFlower.jpg" width="220" height="165" />
+<p>
+  <em>Photos by Greg Hume</em>
+</p>
+<p>
+  This gets more complicated with more content types.
+  It is hard to use text search to match the images without any text.
+  The same goes for videos, podcasts, songs, TV shows, and so on - this is often called multi-modal search.
+  A workaround is to search the object’s textual metadata, like a song’s title, the podcast episode summary,
+  and the image’s <em>alt</em> text on the webpage.
+  Such metadata is only sometimes available and is often too short/imprecise - this does not solve the root problem.
+</p>
+<p>
+  The query can be an image itself or a song, where the users want <em>more like this</em>,
+  in the context of the current song or an image being viewed.
+  This is a problem found in recommender systems.
+</p>
+
+
+
+<h2 id="digital-representations">Digital representations</h2>
+<p>
+  A solution to the synonym or multi-modal problem is to change from matching in the textual to the digital domain.
+  This means transforming text, images, and songs into a set of numbers that describe their <em>meaning</em> of it.
```

Comment:
I think that it's a stretch to use the word meaning here.

Reply:
all right, softened the language to indicate what it is about